lib/include/DOtherSide/DOtherSide.h

```diff
@@ -111,12 +111,48 @@ DOS_API void DOS_CALL dos_qqmlapplicationengine_add_import_path(DosQQmlApplicati
 /// the engine and so it should die with the engine.
 DOS_API DosQQmlContext *DOS_CALL dos_qqmlapplicationengine_context(DosQQmlApplicationEngine *vptr);
 
+/// \brief Calls the QQMLApplicationengine::addImageProvider    
+/// \param vptr The QQmlApplicationEngine
+/// \param vptr_i A QQuickImageProvider, the QQmlApplicationEngine takes ownership of this pointer
+DOS_API void DOS_CALL dos_qqmlapplicationengine_addimageprovider(DosQQmlApplicationEngine *vptr, const char* name, DosQQuickImageProvider *vptr_i);
```

Comment:
For the sake of consistency rename to dos_qqmlapplicationengine_addImageProvider

i know..maybe it's not the best convention but this is how all the other functions look like